README.md

```diff
@@ -1,123 +1,404 @@
+    NAME                      ENDPOINTS                               AGE
+    ep/default-http-backend   10.1.68.2:80                            1h
+    ep/kubernetes             172.31.31.139:6443                      1h
+    ep/microbot               10.1.20.3:80,10.1.68.3:80,10.1.7.4:80   1h
+
+
+#### List the ingress resources
+
+    $ kubectl get ingress
+    NAME               HOSTS                          ADDRESS         PORTS     AGE
+    microbot-ingress   microbot.52.38.62.235.xip.io   172.31.26.109   80        1h
+
+
+When all the pods are listed as Running, the endpoint has more than one host
+you are ready to visit the address in the hosts section of the ingress listing.
+
+It is normal to see a 502/503 error during initial application turnup.
+
+As you refresh the page, you will be greeted with a microbot web page, serving
+from one of the microbot replica pods. Refreshing will show you another
+microbot with a different hostname, as the requests are load balanced through
+out the replicas.
+
+#### Clean up microbot
+
+There is also an action to clean up the microbot applications. When you are
+done using the microbot application you can delete them from the pods with
+one Juju action:
+
+```
+juju run-action kubernetes-worker/0 microbot delete=true
+```
+
+If you no longer need Internet access to your workers remember to unexpose the
+kubernetes-worker charm:
+
+```
+juju unexpose kubernetes-worker
+```
+
+To learn more about
+[Kubernetes Ingress](http://kubernetes.io/docs/user-guide/ingress.html)
+and how to really tune the Ingress Controller beyond defaults (such as TLS and
+websocket support) view the
+[nginx-ingress-controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx)
+project on github.
+
+
+# Scale out Usage
+
+Any of the applications can be scaled out post-deployment. The charms
+update the status messages with progress, so it is recommended to run.
+
+```
+watch -c juju status --color
+```
+
+### Scaling kubernetes-worker
+
+The kubernetes-worker nodes are the load-bearing units of a Kubernetes cluster.
+
+By default pods are automatically spread throughout the kubernetes-worker units
+that you have deployed.
+
+To add more kubernetes-worker units to the cluster:
+
+```
+juju add-unit kubernetes-worker
+```
 
 or specify machine constraints to create larger nodes:
 
-    juju add-unit kubernetes --constraints "cpu-cores=8 mem=32G"
+```
+juju add-unit kubernetes-worker --constraints "cpu-cores=8 mem=32G"
+```
 
 Refer to the
 [machine constraints documentation](https://jujucharms.com/docs/stable/charms-constraints)
-if you want more information about setting larger constraints.
+for other machine constraints that might be useful for the kubernetes-worker units.
+
 
 ### Scaling Etcd
 
 Etcd is used as a key-value store for the Kubernetes cluster. For reliability
-the cluster defaults to three instances in this bundle.
+the bundle defaults to three instances in this cluster.
```

Comment:
The number of etcd units are 1 in this new bundle. We should change 'three' to 'one' here.